### PR TITLE
Enum support

### PR DESCRIPTION
**Pull request by do4gr**

### CODE REVIEW

File: src/connector/postgres/conversion.rs [hunk 1]
```diff
@@ -283,6 +300,10 @@ impl<'a> ToSql for ParameterizedValue<'a> {
                 }
                 _ => float.to_sql(ty, out),
             },
+            ParameterizedValue::Enum(string) => {
+                out.extend_from_slice(format!("{}", string).as_bytes());
```
Comment (pimeys) [inline src/connector/postgres/conversion.rs]: Why you double allocate with `format`, don't you have the ownership to the string, so you can just call `as_bytes` directly?

File: src/connector/postgres/conversion.rs [hunk 2]
```diff
@@ -322,6 +343,10 @@ impl<'a> ToSql for ParameterizedValue<'a> {
                 _ => float.to_sql(ty, out),
             },
             ParameterizedValue::Text(string) => string.to_sql_checked(ty, out),
+            ParameterizedValue::Enum(string) => {
+                out.extend_from_slice(format!("{}", string).as_bytes());
```
Comment (pimeys) [inline src/connector/postgres/conversion.rs]: Same as above.

File: src/connector/postgres/conversion.rs (hunk 1 above)
Comment (do4gr) [reply src/connector/postgres/conversion.rs]: fixed

File: src/connector/postgres/conversion.rs (hunk 2 above)
Comment (do4gr) [reply src/connector/postgres/conversion.rs]: fixed

**Pull request merged**